[Fix #710] Use network_id to identify faucet_url (#715)
XRPLF/xrpl-py: CHANGELOG.md

### Fixed
- Allow empty strings for the purpose of removing fields in DIDSet transaction

> - Refactor the API of `get_faucet_wallet` method. This produces a non-ambiguous URL for Testnet and Devnet faucet hosts.

Reviewer: ```suggestion
- Use `NetworkID` in faucet processing to produce a non-ambiguous URL for faucet hosts.
```
